src/main.cpp

```diff
@@ -2,7 +2,8 @@
 
 int main(int argc, char **argv)
 {
-    auto test = new std::string;
+    auto test = new std::string;  // :)
     std::cout << "Hello xD\n";
+    auto test2 = new std::string;  // :)
```

Comment:
warning: parameter 'argc' is unused [misc-unused-parameters]

```suggestion
int main(int  /*argc*/, char **argv)
```